tests/python/unittest/test_operator.py

```diff
@@ -367,6 +367,108 @@ def test_pow_fn():
     check_symbolic_forward(y, [x], [2**x])
     check_symbolic_backward(y, [x], [np.ones(shape)], [np.log(2) * 2**x])
 
+def test_equal():
```

Comment:
this is due to your lint version is different from the version on test server.
look here http://107.23.36.180:8080/job/mxnet/761/console to fix the ones on the test server.